targets/mgsops/targets.go

```diff
@@ -19,15 +19,21 @@ const version = "3.7.1"
 // nolint: gochecknoglobals
 var executable string
 
+type Prepare mgtool.Prepare
+
+func (Prepare) Sops() {
+	mg.Deps(prepare)
```

Comment:
could we just inline the `prepare` function here?

Reply:
Yep, fixed

Reply:
Might as well inline all prepare functions?